src/plugins/intel_npu/src/plugin/npuw/llm_infer_request.cpp

```diff
@@ -358,7 +363,11 @@ void ov::npuw::LLMInferRequest::update_kvcache_for(
                                            kvcache_desc.num_stored_tokens - num_tokens,
                                            kvcache_desc.num_stored_tokens);
         auto src_tensor = request->get_tensor(out_ports.at(output_name));
-        copy_tensor_by_dim(src_tensor, dst_slice, kv_dim);
+        if (num_tokens == 1) {
+            copy_tensor_by_dim(src_tensor, dst_slice, kv_dim, true);
```

Comment:
as  it is "update_kvcache_for", num_tokens should be always 1 at this stage, isn't it?

Reply:
It is an update for prefill kvcache also, so one time it is chunk_size, another time it is 1